html-relative-links/about-me/about-me.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>About Me</title>
+    <style>
+      img {
+        width: 300px;
+      }
+    </style>
+  </head>
+  <body>
+    <ul>
+      <li><a href="../index.html">Home Page</a></li>
+      <li><a href="../about-me/about-me.html">About Me</a></li>
```

Comment:
When we're in the same directory, we don't need to go out of the directory just to go back in to get to the file You should shorten the relative url here on line 14 otherwise we're forcing the browser to to go through extra steps that are unnecessary.